.gitignore

```diff
@@ -2,3 +2,4 @@
 .vimrc
 *.txt
 sink/logs
+/verify/*secrets*
```

Comment:
The commit log still says "both in tarball and in a Kubernetes Secrets json file.", but that can easily be corrected during merging.